addon/components/ilios-calendar-event-month.js

```diff
@@ -38,6 +38,18 @@ export default CalendarEvent.extend({
   isIlm: notEmpty('event.ilmSession'),
   isOffering: notEmpty('event.offering'),
   clickable: any('isIlm', 'isOffering'),
+
+  recentlyUpdated: computed('event', {
+    get() {
+      const event = this.event;
+      const lastModifiedDate = moment(event.lastModified);
+      const today = moment();
+      const daysSinceLastUpdate = today.diff(lastModifiedDate, 'days');
+
+      return daysSinceLastUpdate < 6 ? true : false;
```

Comment:
Can you make this a property `daysToShowAlert` maybe?  I think we will probably pass it in eventually, but it can be hard coded for now.